ngapp/src/app/core/models/feedbackComment.ts

```diff
@@ -0,0 +1,11 @@
+export class FeedbackComment {
+  range: {
+    index: number,
+    length: number
+  };
+  text: string;
+  constructor(range: {index: number, length: number}, text: string) {
+    this.range = range;
+    this.text = text;
+  }
+}
```

Comment:
What do you think about simplifying this to a `type` instead of a `class`?

```suggestion
export type FeedbackComment {
  range: {
    index: number,
    length: number
  };
  text: string;
}
```

Reply:
Could make this a type, just following all the previous models which are currently defined as classes.  Are types better data structures for small structures like these?  We could change them all to types.